From fb9a7202a7308a1251d0635c10e68486f28bfd3f Mon Sep 17 00:00:00 2001
From: Saurabh Sengar <saurabh.singh@xilinx.com>
Date: Sat, 24 Feb 2018 00:05:02 +0530
Subject: [PATCH 0288/1852] drm: xlnx: mixer: Removing dead code

commit 0aa39228ac637f10817c0f4bdeae79add6823704 from
https://github.com/Xilinx/linux-xlnx.git

Removing unnecessary loop, assigning plane's layer_data to layer_data
pointer.

Signed-off-by: Saurabh Sengar <saurabh.singh@xilinx.com>
Reported-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/gpu/drm/xlnx/xlnx_mixer.c | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/drivers/gpu/drm/xlnx/xlnx_mixer.c b/drivers/gpu/drm/xlnx/xlnx_mixer.c
index b614d1650e73..bcc2bfbceb0b 100644
--- a/drivers/gpu/drm/xlnx/xlnx_mixer.c
+++ b/drivers/gpu/drm/xlnx/xlnx_mixer.c
@@ -2106,13 +2106,6 @@ static int xlnx_mix_plane_create(struct device *dev, struct xlnx_mix *mixer)
 		return ret;
 	}
 
-	/* Init all layers to inactive state in software. An update_plane()
-	 * call to our drm driver will change this to 'active' and permit the
-	 * layer to be enabled in hardware
-	 */
-	for (i = 0; i < mixer_hw->layer_cnt; i++)
-		layer_data = &mixer_hw->layer_data[i];
-
 	mixer->max_width = XVMIX_DISP_MAX_WIDTH;
 	mixer->max_height = XVMIX_DISP_MAX_HEIGHT;
 	if (mixer->hw_logo_layer) {
-- 
2.31.1

